src/metax_api/migrations/0014_catalogrecord_state.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.11 on 2020-02-07 15:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('metax_api', '0013_catalogrecord__access_granter'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='catalogrecord',
+            name='state',
+            field=models.CharField(default='published', help_text='Publishing state (published / draft) of the dataset.', max_length=200),
```

Comment:
Now that the ``choices`` parameter was used in the ``CharField``, I think the allowed values should be present in this definition too (draft, published). You need to:

1) delete this migration file
2) drop db and recreate it
3) run makemigrations again to create new migration file
( 4) run migrate again to get the field back to your current db)